singer_sdk/exceptions.py

```diff
@@ -16,19 +16,19 @@ class FatalAPIError(Exception):
     """Exception raised when a failed request should not be considered retriable."""
 
 
-class InvalidStreamSortException(Exception):
+class InvalidStreamSortException(Exception):  # noqa: N818
```

Comment:
I'd be fine seeing `N818` disabled globally. There's nothing wrong with ending an exception's name in `Exception` imo.